cmd/fleet/cron.go

```diff
@@ -353,7 +353,6 @@ func checkNVDVulnerabilities(
 			VulnPath:           config.DatabasesPath,
 			CPEDBURL:           config.CPEDatabaseURL,
 			CPETranslationsURL: config.CPETranslationsURL,
-			CVEFeedPrefixURL:   config.CVEFeedPrefixURL,
```

Comment:
@noahtalerman

I believe we have the OK from product to not require support for syncing old feeds.
Starting in 4.41.0 Fleet will use the NVD 2.0 API to get CVE information from nvd.nist.gov and not support getting the CVEs information from nvd.nist.gov or a given URL using the legacy format.

This means that this `cve_feed_prefix_url` config will be deprecated in 4.41.0
https://github.com/fleetdm/fleet/blob/8054e784e9ddf53ff86ca3fd5f26b9ac5495cf99/docs/Configuration/fleet-server-configuration.md?plain=1#L2364-L2380

Let me know if we are all on the same page.

Reply:
@getvictor So the decision is that we will keep this flag (and use legacy sync) just in case as plan B, right?

Reply:
Makes sense we're deprecating it. 

@rachaelshaw is this the messaging we usually use in the docs for depreciated config options?

Should we just remove it from the docs?

Reply:
We are actually not deprecating this flag because it serves as a fallback in case we need to move the CVE syncing and "pre-cooking" of data to our fleetdm/nvd or similar repository.

Reply:
We also needed this to unblock the CI vulnerability tests which currently use the legacy feed (otherwise CI will take a lot of time to run). Am creating a separate issue to amend CI to use the new API 2.0 in a way that it doesn't take long and can exercise the new vuln code.

Reply:
We want to be able to use this flag for contributors for the initial sync, but then use API for subsequent sync. We have an issue for this: https://github.com/fleetdm/fleet/issues/15231

Reply:
Ah, ok. Maybe we move it to the contributors docs then? 

Sounds like we don't want customers/users to run into it now.